docs/architecture/grpc-for-wcf-developers/types-of-rpc.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Types of RPC
+description: gRPC for WCF Developers | Types of RPC
+author: markrendle
+ms.date: 09/02/2019
+---
+
+# Types of RPC
+
+As a WCF developer you are probably used to dealing with the following types of RPC:
+
+- Request/Reply
+- Duplex:
+  - One way Duplex
+  - One way Duplex with Session
+  - Full Duplex
+
+It is possible to map these RPC types fairly naturally to existing gRPC concepts and this chapter will look at each of these areas in turn. Similar examples will be explored in much greater depth in Chapter 5.
+
+| WCF | gRPC |
+| --- | ---- |
+| Regular request/reply | Unary |
+| Duplex Service with OneWay operations on client callback interface | Server-streaming |
```

Comment:
nit: we haven't been hyponating server streaming or client streaming in our docs